assignments/objects.js

```diff
@@ -49,16 +86,39 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  child: {
+    name: "George",
+    age: 50,
+    child: {
+      name: "Sam",
```

Comment:
Great job on working on stretch goals!
You nailed them.

I have to say though I don't really like how you called in the same way the two child objects.
I find it a bit confusing. On line 117-118 would be probably more readable just to have it 
`console.log(parent.child.grandchild.age);`